### PR TITLE
COL-775, user.last_activity with smart date format

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-775

Possible date formats:
* 'Today'
* 'Yesterday'
* 'dddd' (if in past 7 days)
* 'MMMM D, YYYY'  (e.g., April 12, 2017)

### CODE REVIEW

File: public/app/dashboard/profileController.js [hunk 1]
```diff
@@ -111,34 +111,30 @@
...
+      $scope.user = user;
+      $scope.showEngagementIndexBox = me.course.engagementindex_url && ($scope.isMyProfile || me.is_admin || (user.share_points && me.share_points));
+      determineRank(user);
+      if ($scope.isMyProfile || me.is_admin) {
+        getUserActivity(user.id);
+      }
+      sortFeaturedAssets($scope.sortAssetsBy);
+      sortCommunityAssets($scope.sortCommunityBy);
+      $scope.user.hashtags = ['#badminton', '#bridge', '#break-dancing'];
+    };
```
Comment (johncrossman) [inline public/app/dashboard/profileController.js]: De-dupification.

**Comment by johncrossman** [comment]
Wait to merge. One amendment coming soon.

**Comment by johncrossman** [comment]
Amendment is in.

**Comment by paulkerschen** [comment]
Never say never indeed.

**Pull request merged**